src/api/api.js

```diff
@@ -0,0 +1,148 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const logger = require('../logger');
+
+const ApiHandler = require('./api-handler');
+
+/**
+ * Ghost Pinbot API implementation
+ */
+class Api {
+  constructor(apiHandler) {
+    this.port = process.env.PORT || 8081;
+    this.handler = apiHandler || new ApiHandler();
+  }
+
+  /**
+   * Starts the server
+   * @return {Promise<void>}
+   */
+  async start() {
```

Comment:
Doesn't seem this method needs to be `async` as there's no `await` statement.

I think it would be useful to return a promise though that would resolve once the server is listening or reject if it fails to, ex:

```js
start() {
  return new Promise((resolve, reject) => {
    ...
    this.app.listen(this.port, (err) => {
      if (err) {
        reject(err)
      } else {
        logger.info(`Server started on port ${this.port}`)
        resolve()
      }
    });
  })
}
```

Reply:
👍